CONTRIBUTING.adoc

```diff
@@ -103,74 +109,74 @@ so there is no strict need to run full test suites before proposing a pull reque
 
-```sh
+[source,shell]
+----
 cd war; yarn test
-```
+----
 
-## Proposing Changes
+== Proposing Changes
 
 The Jenkins project source code repositories are hosted at GitHub.
-All proposed changes are submitted, and code reviewed, using a [GitHub pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) process.
+All proposed changes are submitted, and code reviewed, using a https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests[GitHub pull request] process.
 
 To submit a pull request:
 
-1. Commit your changes and push them to your fork on GitHub.
-   It is a good practice is to create branches instead of pushing to master.
-2. In the GitHub Web UI, click the _New Pull Request_ button.
-3. Select `jenkinsci` as _base fork_ and `master` as `base`, then click _Create Pull Request_.
-   - We integrate all changes into the master branch towards the Weekly releases.
-   - After that, the changes may be backported to the current LTS baseline by the LTS Team.
-     Read more about the [backporting process](https://www.jenkins.io/download/lts/).
-4. Fill in the Pull Request description according to the [proposed template](.github/PULL_REQUEST_TEMPLATE.md).
-5. Click _Create Pull Request_.
-6. Wait for CI results/reviews, process the feedback.
-   - If you do not get feedback after 3 days, feel free to ping `@jenkinsci/core-pr-reviewers` in the comments.
-   - Usually we merge pull requests after 2 approvals from reviewers, no requested changes, and having waited some more time to give others an opportunity to provide their feedback.
-     See [this page](/docs/MAINTAINERS.adoc) for more information about our review process.
+. Commit your changes and push them to your fork on GitHub. It is a good practice is to create branches instead of pushing to master.
+. In the GitHub Web UI, click the _New Pull Request_ button.
+. Select `jenkinsci` as _base fork_ and `master` as `base`, then click _Create Pull Request_.
+* We integrate all changes into the master branch towards the Weekly releases.
+* After that, the changes may be backported to the current LTS baseline by the LTS Team.
+  Read more about the https://www.jenkins.io/download/lts/[backporting process].
+. Fill in the Pull Request description according to the link:.github/PULL_REQUEST_TEMPLATE.md[proposed template].
+. Click _Create Pull Request_.
+. Wait for CI results/reviews, process the feedback.
+* If you do not get feedback after 3 days, feel free to ping `@jenkinsci/core-pr-reviewers` in the comments.
+* Usually we merge pull requests after 2 approvals from reviewers, no requested changes, and having waited some more time to give others an opportunity to provide their feedback.
+  See link:/docs/MAINTAINERS.adoc[this page] for more information about our review process.
```

Comment:
```suggestion
  See link:MAINTAINERS.adoc[this page] for more information about our review process.
```
(In case https://github.com/jenkinsci/jenkins/pull/7528 is merged first)